pim-tool-back/src/main/java/vn/elca/training/model/exception/ProjectNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package vn.elca.training.model.exception;
+
+public class ProjectNotFoundException extends RuntimeException{
```

Comment:
Should business exception be checked or unchecked exception?

Reply:
unchecked exceptions, business exception is an exception that occurs due to a problem in the business logic of the application, this type of exception is not caused by a programming error, and therefore it is not necessary for the caller to handle it
If business exceptions were checked exceptions, then the caller would be forced to handle them, even though there is nothing they can do to recover from the exception. This would make the code less readable and maintainable.